index.html

```diff
@@ -0,0 +1,1305 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Page Title</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <link rel='stylesheet' type='text/css' media='screen' href='css/styles.css'>
+  <link rel='stylesheet' type='text/css' media='screen' href='css/mediaquery.css'>
+  <script src="https://kit.fontawesome.com/03774beff9.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header class="container-fluid navbar navbar-expand-lg navbar-light d-lg-flex d-lg-justify-content-between w-xs-100">
```

Comment:
Please use bootstrap classes when possible. Your page doesn't look like the original one. You should apply a bootstrap class to the `<body>` tag (perhaps some col-* and container-fluid) to make it less wide and achieve the desired look (you should see the gray background color when the screen is larger)